Automattic/jetpack
# AI Assistant: Add UsageBar component

Part of #33359.

## Proposed changes:


* Add UsageBar component to represent the usage
* Add a sample version of it on the UsagePanel so we can see how it looks
* Add sample usage text to the UsagePanel as well, to complete the design

You can check the Figma designs on the i6 post about the feature: p1HpG7-oMv-p2.

The upgrade button will be added on a follow up. The real data will be plugged on a follow up.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Open the Jetpack sidebar
* Look for the AI Assistant section and open it:

<img width="300" alt="Screenshot 2023-10-19 at 18 05 48" src="https://github.com/Automattic/jetpack/assets/6760046/65d1d449-9aad-4480-9bba-b38c6c04fff1">

* Check the usage panel there and compare with the provided design:

<img width="250" alt="Screenshot 2023-10-19 at 18 04 39" src="https://github.com/Automattic/jetpack/assets/6760046/b5fae23f-16eb-4569-9cae-a89953e21eb6">
